src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -71,6 +74,18 @@ data class ReportingOfficer(
   val username: String,
 )
 
+@Schema(description = "witnesses")
+data class MigrateWitness(
+  @Schema(description = "first name")
+  val firstName: String,
+  @Schema(description = "last name")
+  val lastName: String,
+  @Schema(description = "created by username")
+  val createdBy: String,
+  @Schema(description = "type of witness")
```

Comment:
given we do not support prisoners as witness, it would be preferable if the migration service maps witnesses to our types